# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/service/VRService.cpp
# Commit: 1d9384906287
# Full Hash: 1d93849062870a8d492eeef2150ecea174cb6223
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-26 21:12:53
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r f5ebdcca640d -r 1d9384906287 gfx/vr/service/VRService.cpp
--- a/gfx/vr/service/VRService.cpp	Sun May 26 14:29:56 2019 +0000
+++ b/gfx/vr/service/VRService.cpp	Sun May 26 14:30:14 2019 +0000
@@ -52,7 +52,7 @@
 already_AddRefed<VRService> VRService::Create() {
   MOZ_ASSERT(NS_IsMainThread());
 
-  if (!gfxPrefs::VRServiceEnabled()) {
+  if (!StaticPrefs::VRServiceEnabled()) {
     return nullptr;
   }
 
@@ -73,7 +73,7 @@
 #if defined(XP_WIN)
       mMutex(NULL),
 #endif
-      mVRProcessEnabled(gfxPrefs::VRProcessEnabled()) {
+      mVRProcessEnabled(StaticPrefs::VRProcessEnabled()) {
   // When we have the VR process, we map the memory
   // of mAPIShmem from GPU process.
   // If we don't have the VR process, we will instantiate